app/javascript/packages/document-capture/components/documents-step.jsx

```diff
@@ -46,7 +39,16 @@ function DocumentsStep({
   const { isMobile } = useContext(DeviceContext);
   const { isLastStep } = useContext(FormStepsContext);
   const { flowPath } = useContext(UploadContext);
-  const { notReadySectionEnabled, exitQuestionSectionEnabled } = useContext(FeatureFlagContext);
+  const { notReadySectionEnabled, exitQuestionSectionEnabled, selfieCaptureEnabled } =
+    useContext(FeatureFlagContext);
+
+  /**
+   * Sides of document to present as file input.
+   *
+   * @type {DocumentSide[]}
+   */
+  const documentSides = selfieCaptureEnabled ? ['front', 'back', 'selfie'] : ['front', 'back'];
```

Comment:
🚀

Reply:
I considered a few ways to add a comment and none of them seemed to add much clarity to me. I think that doing the work in [LG-11653](https://cm-jira.usa.gov/browse/LG-11653) instead will address this?